src/server.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const http = require('http');
+
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  const normalizeUrl = new URL(req.url, `http://${req.headers.host}`);
+
+  const parts = normalizeUrl.pathname.slice(1).split('/');
+  const query = Object.fromEntries(normalizeUrl.searchParams.entries());
```

Comment:
it returns only last value of key) but for this task it's enough, but be carefull and keep in mind that 
get - returns first key value by key,
getAll - return all values by key
obj.fromENtries - all key with last values